.github/workflows/6_build.yml

```diff
@@ -0,0 +1,58 @@
+name: Build extension
+
+on:
+  workflow_dispatch:
+    inputs:
+      npm_channel:
+        description: 'Prisma CLI NPM Channel'
+        required: true
+      extension_version:
+        description: 'VSCode version'
+        required: true
+      branch:
+        description: 'Branch to run run this workflow on'
+        required: true
+
+jobs:
+  build:
+    name: Build extension
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.branch }}
+      - name: Install Dependencies
+        run: npm install
+      - name: Compile
+        run: npm run compile
+      - name: Replace Readme
+        run: npm run change_readme  ${{ github.event.inputs.npm_channel }}
+      - name: Build vsix 
+        run: cd packages/vscode && npm run package
+      - name: Get tag and asset name
+        id: names
+        run: npm run get_tag_name ${{ github.event.inputs.npm_channel }} ${{ github.event.inputs.extension_version }}
+      - name: create a release
+        id: create_release
+        uses: actions/create-release@v1
+        env: 
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ steps.names.outputs.tag_name }}
+          release_name: ${{ steps.names.outputs.tag_name }}
+      - name: Upload Release Asset
+        id: upload-release-asset
+        uses: actions/upload-release-asset@v1
+        env: 
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: ${{github.workspace}}/${{ steps.names.outputs.asset_name }}-${{ github.event.inputs.extension_version }}.vsix
+          asset_name: ${{ steps.names.outputs.asset_name }}-${{ github.event.inputs.extension_version }}.vsix
+          asset_content_type: application/zip     
+      - name: Trigger next workflow to publish extension
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: Publish
+          token: ${{ secrets.GH_TOKEN }}
+          inputs: '{ "asset_name": "${{ steps.names.outputs.asset_name }}-${{ github.event.inputs.extension_version }}.vsix", "tag_name": "${{ steps.names.outputs.tag_name }}" }'
```

Comment:
Should this maybe already be a variable further up that is just used here (instead of duplicating the file name createion from line 51 again)?